cairo_programs/cairo-1-contracts/should_skip_squash_loop.cairo

```diff
@@ -0,0 +1,10 @@
+#[contract]
+mod ShouldSkipSquashLoop {
+    use dict::Felt252DictTrait;
+
+    #[external]
+    fn should_skip_squash_loop() {
+        let x = felt252_dict_new::<felt252>();
+        x.squash();
+    }
+}
```

Comment:
add endline

Reply:
Maybe i am wrong here, but can we return `x.squash()` so we can check it in the test